ShopifySharp/Entities/FulfillmentRequestOrderLineItem.cs

```diff
@@ -0,0 +1,23 @@
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// An object representing a Shopify fulfillment request order line items.
+    /// </summary>
+    public class FulfillmentRequestOrderLineItem
+    {
+        /// <summary>
+        /// The ID of the fulfillment order line item.
+        /// </summary>
+        [JsonProperty("id")]
+        public long? Id { get; set; }
```

Comment:
Instead of adding the `Id` property directly to this new class, the class itself should extend the base `ShopifyObject` which contains the `Id` property:

```cs
public class FulfillmentRequestOrderLineItem : ShopifyObject
{
        /// <summary>
        /// The total number of units to be fulfilled.
        /// </summary>
	[JsonProperty("quantity")]
	public long? Quantity { get; set; }
}
```